### ISSUE TITLE
tooltip: shadow is drawn twice in macOS

### ISSUE THREAD

**Issue by vlsi** [weisJ/darklaf#102]
Tooltip renders shadow twice: first darklaf paints its shadow, then macOS paints its own (because heavy-weight windows drop shadow in macOS by default)

d43a3aa4be51a6e84c08b613e4a790c579286bcb (double shadow):

<img width="247" alt="Снимок экрана 2020-03-20 в 2 30 05" src="https://user-images.githubusercontent.com/213894/77124024-cef13c00-6a52-11ea-8961-a869aea81e4f.png">

With `rootPane.putClientProperty("Window.shadow", false);` (== only darklaf shadow)

<img width="201" alt="Снимок экрана 2020-03-20 в 2 30 28" src="https://user-images.githubusercontent.com/213894/77124037-db759480-6a52-11ea-95e9-87c7f2fdd9f0.png">

I don't know how to disable darklaf shadow, however it would be interesting to be able to use native shadow only.

Just in case, here's a menu (which uses native shadow only):
<img width="209" alt="Снимок экрана 2020-03-20 в 2 32 38" src="https://user-images.githubusercontent.com/213894/77124133-1c6da900-6a53-11ea-818c-856b6bcd83d7.png">

**Comment by weisJ** [comment]
How does the tooltip look when you comment out the part in `DarkToolTipBorder` where the shadow border gets drawn? Does the native shadow respect the shape of the non transparent part?

**Comment by vlsi** [comment]
```java
//shadowBorder.paintBorder(c, g, x + bw, y + bw + off, width - 2 * bw, height - 2 * bw - off);
```

<img width="247" alt="Снимок экрана 2020-03-20 в 2 39 26" src="https://user-images.githubusercontent.com/213894/77124491-0c09fe00-6a54-11ea-9579-25f8d56897ec.png">

**Comment by weisJ** [comment]
In this case I’ll simply just add a flag for whether to paint the shadow. On macOS this can be set to `false` then.

**Comment by weisJ** [comment]
Fixed in 44e7399ad6e835dd2c0f9e6cb59cae1feb7f8a83
